cypress.json

```diff
@@ -20,6 +20,8 @@
     "VISBUILDER_ENABLED": true,
     "DATASOURCE_MANAGEMENT_ENABLED": false,
     "ML_COMMONS_DASHBOARDS_ENABLED": true,
-    "WAIT_FOR_LOADER_BUFFER_MS": 0
+    "WAIT_FOR_LOADER_BUFFER_MS": 0,
+    "_COMMENT_FOR_NO_COMMAND_LOG_": "Set NO_COMMAND_LOG to 1 to reduce memory usage, or some test cases will hang there forever.",
+    "NO_COMMAND_LOG": 1
```

Comment:
By disable the command log, if there has some test case failure, does the video or screenshot know which one is failed?

Reply:
Just check the screenshot of fail test case, it looks like below, a little bit hard for trouble shooting
![image](https://github.com/opensearch-project/opensearch-dashboards-functional-test/assets/113890546/d337dd2f-23ca-4854-bc1f-96e0da8b30d3)

Reply:
You are right, let me see if there is other option.